crates/holochain_conductor_api/src/admin_interface.rs

```diff
@@ -633,21 +633,12 @@ impl ExternalApiWireError {
 #[serde(rename_all = "snake_case")]
 /// Filter for [`AdminRequest::ListApps`].
 ///
-/// App Status is a combination of two pieces of independent state:
-/// - Enabled/Disabled, which is a designation set by the user via the conductor interface.
-/// - Running/Stopped, which is a fact about the reality of the app in the course of its operation.
+/// Apps can be either enabled or disabled, set by the user via the conductor interface.
 pub enum AppStatusFilter {
-    /// Filter on apps which are Enabled, which can include both Running and Paused apps.
+    /// Filter on apps which are Enabled.
     Enabled,
-    /// Filter only on apps which are Disabled.
+    /// Filter on apps which are Disabled.
     Disabled,
-    /// Filter on apps which are currently Running (meaning they are also Enabled).
-    Running,
-    /// Filter on apps which are Stopped, i.e. not Running.
-    /// This includes apps in the Disabled status, as well as the Paused status.
-    Stopped,
```

Comment:
Ahh, I remember now. So stopped is a composite state and we should be removing references to "stopped" in favor of "disabled"

Reply:
Actually, I'm seeing that the [StoppedAppReason](https://github.com/holochain/holochain/blob/a7ca376ac98d709c0594b52952696fe22ef54742/crates/holochain_types/src/app.rs#L1097-L1105) contains the `AwaitingMemproofs` variant which I'm not sure we can conflate with `Disabled` completely. But I'm also not entirely sure how `StoppedAppReason` relates to `AppStatusFilter`.

Reply:
@matthme Did you mean to say "should not be"?